eth_bridge/eth/contracts/EthProxy.sol

```diff
@@ -27,19 +35,14 @@ contract EthProxy {
         terabethiaCore.consumeMessage(CANISTER_ADDRESS, payload);
 
         // withdraw eth
-        require(
```

Comment:
Any reason this condition was removed?

Reply:
Yes, cause it is useless, its gonna fail anyways